src/dpe.py

```diff
@@ -215,17 +216,15 @@ def run(self, net):
 
     total_tiles = count_tiles(model_path) - 2
     print(total_tiles)
-
-    if(args.tile != -1):
-        total_tiles = int(args.tile)
-
```

Comment:
Don't remove the past additions (probably get the most recent master), before merging your local commit, and then followed by pull request.